statsmodels/sandbox/regression/ols_anova_original.py

```diff
@@ -327,7 +328,7 @@ def dropname(ss, li):
 # -------------------
 
 # read data set and drop rows with missing data
-dta = np.genfromtxt("dftest3.data", dt_b, missing=".", usemask=True)
+dta = pd.read_csv("dftest3.data", header=None, na_values=".").values
```

Comment:
this will not work. I guess the code will raise an exception.
This is old sandbox code, I guess, without unit tests.

We could leave this for after this PR.
My guess is that the code up to and including line 337 can be replace by `dta.dropna()`

But it is likely not worth the effort to rescue this module.